modules/tinymce/src/plugins/table/main/ts/ui/ButtonToggleUtils.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Singleton } from '@ephox/katamari';
+import Editor from 'tinymce/core/api/Editor';
+import { Toolbar } from 'tinymce/core/api/ui/Ui';
+
+const onSetupToggle = (editor: Editor, styleName: string, styleValue: string) => {
+  return (api: Toolbar.ToolbarToggleButtonInstanceApi) => {
+    const boundCallback = Singleton.unbindable();
+
+    const init = () => {
+      const value = {
+        value: styleValue
+      };
+
+      api.setActive(editor.formatter.match(styleName, value));
```

Comment:
This is overly verbose and can be inlined:
```suggestion
      api.setActive(editor.formatter.match(styleName, { value: styleValue }));
```